### PR TITLE
Fix Preview

### LINKED ISSUE #1828: Preview section overlaps with footer links
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

Preview section overlaps with footer links, when page is scrolled down to bottom.

screenshot from 2018-10-17 12-00-45

### PR DESCRIPTION
**Pull request by sk9331657**
Fixes #1828 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-FIx overlapping Preview

### Screenshots for the change

ezgif com-video-to-gif 3

### CODE REVIEW

**Comment by sk9331657** [comment]
@mariobehling  PLease review this also.

**Comment by sk9331657** [comment]
@ParthS007  @gabru-md  @yashLadha  Reviews.

**Comment by sk9331657** [comment]
@gabru-md  Please review this also.

File: frontend/app/components/create-badge.js [hunk 1]
```diff
@@ -7,15 +7,17 @@ export default Component.extend({
...
       var card = EmberObject.$(document)[0].getElementsByClassName('six wide column preview');
       if (card.length != 0) {
-        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && this.scrollY < 1278) {
+        if (e.type === 'scroll' && this.scrollY >= 137 && this.window.innerWidth >= 764 && !(EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height() - 100))) {
           card[0].style.position = 'fixed';
           card[0].style.right = '0';
           card[0].style.top = '0';
-        } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && this.scrollY >= 1278) {
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
+
```
Comment (ParthS007) [inline frontend/app/components/create-badge.js]: Please remove this Empty line.

Comment (sk9331657) [reply frontend/app/components/create-badge.js]: @ParthS007  Done.

**Comment by sk9331657** [comment]
@gabru-md  @ParthS007  Please review.

**Comment by sk9331657** [comment]
@ParthS007  Please review @yashLadha

**Comment by sk9331657** [comment]
@fossasia/badgeyay-admins  @ParthS007  @gabru-md Please review,

**Comment by sk9331657** [comment]
@abishekvashok  @ParthS007  @gabru-md  @yashLadha  @jajodiaraghav  Please review.

**Comment by sk9331657** [comment]
@abishekvashok  Please review,

File: frontend/app/components/create-badge.js [hunk 2]
```diff
@@ -7,15 +7,16 @@ export default Component.extend({
...
           card[0].style.top = '0';
-        } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && this.scrollY >= 1278) {
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
+        } else if (e.type === 'scroll' && this.window.innerWidth >= 764 && (EmberObject.$(window).scrollTop() + EmberObject.$(window).height() > (EmberObject.$(document).height() - 100))) {
           card[0].style.position = 'fixed';
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '570px';
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '530px ';
         } else {
-          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px';
+          EmberObject.$(document)[0].getElementsByClassName('bgImg ui raised segment')[0].style.height = '600px ';
```
Comment (gabru-md) [inline frontend/app/components/create-badge.js]: remove the extra space after `px`?

**Comment by sk9331657** [comment]
@gabru-md  Review now.

**Comment by mariobehling** [comment]
Please resolve conflicts.

**Comment by sk9331657** [comment]
@mariobehling  Done.

**Comment by sk9331657** [comment]
@mariobehling @gabru-md  Please review this also.

**Pull request merged**